engines/enginetest/display.go

```diff
@@ -112,6 +116,44 @@ func (c *DisplayTestCase) TestDisplays() {
 	}
 }
 
+// TestKillDisplay opens a display and test that it is closed if Sandbox.Kill()
+// is called.
+func (c *DisplayTestCase) TestKillDisplay() {
+	debug("## TestKillDisplay")
+	r := c.newRun()
+	defer r.Dispose()
+	r.NewSandboxBuilder(c.Payload)
+	r.StartSandbox()
+
+	// List displays
+	displays, err := r.sandbox.ListDisplays()
+	nilOrPanic(err, "Failed to ListDisplays")
```

Comment:
yeah, and we should carry the `*testing.T` variable into all these methods... But we haven't done this yet...

I think we should do it in one big swoop for `enginetest`... pete did it some places, but not in enginetests